### PR TITLE
Trev8939/feature render mode develop

### PR DESCRIPTION
**Pull request by TADraeseke**
Sample which shows some of the differences between static and dynamic rendering modes.

https://devtopia.esri.com/runtime/java-android-sdk/issues/1407

### CODE REVIEW

File: java/feature-layer-rendering-mode-map/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            android:text="@string/renderingmode_static"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:id="@+id/staticTextView"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_marginTop="8dp"
+            android:layout_marginStart="8dp"
+            app:layout_constraintLeft_toLeftOf="parent"
+            android:textStyle="bold"
+            android:textColor="@android:color/background_light"/>
```
Comment (doneill) [inline java/feature-layer-rendering-mode-map/src/main/res/layout/activity_main.xml]: Change to `background_dark` for better visibility

File: java/feature-layer-rendering-mode-map/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            android:text="@string/renderingmode_dynamic"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:id="@+id/dynamicTextView"
+            app:layout_constraintBottom_toBottomOf="parent"
+            android:layout_marginBottom="32dp"
+            android:layout_marginStart="8dp"
+            app:layout_constraintLeft_toLeftOf="parent"
+            android:textStyle="bold"
+            android:textColor="@android:color/background_light"/>
```
Comment (doneill) [inline java/feature-layer-rendering-mode-map/src/main/res/layout/activity_main.xml]: Change to `background_dark` for better visibility

**Comment by TADraeseke** [comment]
Changes address. Thanks @doneill!

**Comment by doneill** [comment]
@TADraeseke updates look good to merge.

**Pull request merged**